### DISCUSSION TITLE
ClockBoard for 30 world cites

### DISCUSSION THREAD

**Discussion by mtennekes** [zonebuilders/zonebuilder#32] [Show and tell]
![cities1_15](https://user-images.githubusercontent.com/2444081/109546731-dd513000-7aca-11eb-94e3-e1c60024ea18.png)
![cities16_30](https://user-images.githubusercontent.com/2444081/109546760-e510d480-7aca-11eb-83cc-c92c582cefeb.png)

Info:

* Script: https://github.com/zonebuilders/zonebuilder/blob/master/sandbox/Dutch_city_population.R
* The blue raster data is population data from WorldPop.
* The red lines are administrative borders (which I added for some cities)

Some observations / points of discussion:

* Regarding admin borders 1) although London and Paris are similar in scale, the city borders of Paris are much tighter, in other words, it doesn't make sense to do statistical comparison analysis of London and Paris using admin borders. 2) Some admin borders do not capture the population density well; look at Mexico.
* Where exactly is the city centre? For the plots, I've used OpenStreetMap nomitatim (`tmaptools::geocode_OSM`) and the metropolitan data from naturalearthdata (the `metro` dataset of `tmap`). For each city, I've selected the best one (as far as I could judge). Does anyone know a good dataset for that?
* As you can see, this plot compares 30 cities of different scales. It really shows how large (and how small) some cities are. For smaller cities, it makes sense to recreate this plot with fewer circles.

**Comment by Robinlovelace** [comment]
This is so cool @mtennekes thanks for making this happen, ideal for the paper!

**Comment by Robinlovelace** [comment]
In terms of discussion points...


- Regarding admin borders 1) although London and Paris are similar in scale, the city borders of Paris are much tighter, in other words, it doesn't make sense to do statistical comparison analysis of London and Paris using admin borders.

Agreed, this is a strength of our approach. Great discussion point.

- Where exactly is the city centre? For the plots, I've used OpenStreetMap nomitatim (tmaptools::geocode_OSM) and the metropolitan data from naturalearthdata (the metro dataset of tmap). For each city, I've selected the best one (as far as I could judge). Does anyone know a good dataset for that?

Seems OK to me and good enough for now. I suggest trying Google's geocoding service (e.g. in `stplanr::geo_code()` ; ) if they are really bad but good enough for our purposes is my thinking.
 
- As you can see, this plot compares 30 cities of different scales. It really shows how large (and how small) some cities are. For smaller cities, it makes sense to recreate this plot with fewer circles.

Agreed.

**Comment by mtennekes** [comment]
Thx, I'll add the results of Google's geocoding to the list of sources.

This example illustrates that one of our challenges is how to decide how many circles are recommended per city. I like your term 'sphere of influences' (mentioned in https://github.com/zonebuilders/zonebuilder/discussions/30). That should be leading, although hard to quantify. Maybe we can come up with something like a "buzz"yness indicator per grid cell and try fit Gaussian distributions (spheres) that represent the cities. But that is something for another paper;-)

I've also included Amsterdam, not because of its size (it is not even a town compared to most of the other cities), but it is a good example of an urban area. The whole area is called the Randstad and has a total population of about 8 million. You may recognize Utrecht, which is the spot in H5. The Hague and Rotterdam are just outside the Clockboard (J8 and K7 respectively).
So the center of the Randstad is somewhere in the middle of these four cities (say G6). The public transport is generally very good, so it takes less than 1,5 hours to travel from one end to the other. It takes much longer in most other cities. However, I do not think it is a good idea to use the ClockBoard for this whole area (Randstad), simply it does not fit.

**Reply by Robinlovelace** [reply]
I think the urban areas in The Netherlands are good: if it can work there in the highly polycentric conurbations it can surely work anywhere! Let me know next steps on the paper.

**Comment by Robinlovelace** [comment]
Quick thought on this @mtennekes: remove the 2 outer rings in all cases. That will mean some cities have boundaries going outside and that will be fine.

**Comment by mtennekes** [comment]
Great suggestion @Robinlovelace :
![cities7c_1_15](https://user-images.githubusercontent.com/2444081/110200695-75e70780-7e5f-11eb-880f-79f565c3ca80.png)
![cities7c_16_30](https://user-images.githubusercontent.com/2444081/110200699-7a132500-7e5f-11eb-8c2c-b5c4035dea9c.png)

**Reply by Robinlovelace** [reply]
Looking good! improvement 4 sure.

**Comment by mtennekes** [comment]
Next steps for the paper: I think it would be good to focus on the whole paper until applications. So I mean the intro (which is almost there), the method section, and probably something like a 'face validity' section in which we can use these charts; we can discuss how well the CB fits in certain cities and where potential problems are.

**Reply by Robinlovelace** [reply]
OK, will take a look today. I may call if I have questions.

**Reply by mtennekes** [reply]
Please do. This month I am extremely busy with other projects, so I expect to have time in April. We should aim to finish the paper by than.

**Comment by mtennekes** [comment]
B.t.w. the arrangements of Paris have a 'zonebuiler'-esque structure, but the labeling is not easy for navigation purposes:
![image](https://user-images.githubusercontent.com/2444081/110203903-bd29c400-7e70-11eb-8111-e0f31416f7a5.png)
